setup.py

```diff
@@ -38,11 +35,6 @@ def run(self):
         _sdist.run(self)
 
 
-class install(_install):
```

Comment:
Yeah, this is why I tried pinging you in https://github.com/ua-parser/uap-python/pull/27#issuecomment-147185573

So `install` step shouldn't be doing this work. Install is what gets run over an `sdist` bundle.

Stepping back, if we were to publish to pypi as we've been doing for years, we'd run `setup.py sdist upload`, which will create the `tar.gz`, and upload to pypi. Now, if we want to install, we can run `pip install ./dist/whatever.tar.gz` to install directly from the tarball.

This effectively extracts the tarball, and runs `setup.py install`. The problem is that we can't call `install_regexes` at this point since it's not a source. `install` already expects the artifacts to be bundled up.

I'm not convinced that we can actually support your use case without breaking the 90% use case of installing from pypi.

Unless we make `install` fail gracefully. That way it would work if the artifacts exist, and noop, or run `install_regexes` if they don't, for your case of trying to install directly from GitHub.

Does that make sense?